stock_picking_invoice_link/init_hook.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2023 ForgeFlow, S.L.
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+import logging
+from collections import deque
+
+from odoo import SUPERUSER_ID, api
+from odoo.tools import sql
+
+_logger = logging.getLogger(__name__)
+
+
+def _create_delivery_count(cr):
+    """Extracted to reuse in migration script. TODO: It can be merged into a sole
+    method in v16"""
+    _logger.info("Creating column: delivery_count for table account_move")
+    cr.execute(
+        """
+    ALTER TABLE account_move
+        ADD COLUMN IF NOT EXISTS delivery_count INTEGER;
```

Comment:
Please take the occasion to rename it to `picking_count`, as it's totally incorrect (and more with `stock_picking_invoice_link_purchase`.